KCPNet.cpp

```diff
@@ -513,13 +519,9 @@ void KCPNetServer::netWorkerServer(const std::function<void(const char *, size_t
     while (true) {
         auto[received_bytes, status] = mKissnetSocket.recv(receiveBuffer);
         if (!received_bytes || status != kissnet::socket_status::valid) {
-            KCP_LOGGER(false, LOGG_NOTIFY, "serverWorker quitting")
+            KCP_LOGGER(false, LOGG_NOTIFY, "serverWorker quitting");
             break;
         }
-        if (status == kissnet::socket_status::non_blocking_would_have_blocked) {
```

Comment:
The above if-clause breaks the loop if status != valid. Therefor I could remove these lines.